src/build.ts

```diff
@@ -216,7 +216,10 @@ async function _build(
         .filter(Boolean)
         .sort() as unknown as Set<string>,
     )) {
-      if (cleanedDirs.some((c) => dir.startsWith(c))) {
+      if (
+        dir === options.rootDir ||
```

Comment:
I think maybe we can use normalize + check startsWith to make sure is also not above rootDir

Reply:
Done ✅ 

The entry is already normalised, here:

https://github.com/unjs/unbuild/blob/01584da5646a9df68f064cbbd277dbb94c46673a/src/build.ts#L186-L187